cli/main.go

```diff
@@ -45,8 +45,7 @@ func buildVersionString() string {
 	}
 	fmt.Fprint(versionsWriter, "\n")
 	fmt.Fprintf(versionsWriter, "reference values for %s platform:\n", platforms.AKSCloudHypervisorSNP.String())
-	fmt.Fprintf(versionsWriter, "\truntime handler:\tcontrast-cc-%s\n", manifest.TrustedMeasurement[:32])
-	fmt.Fprintf(versionsWriter, "\tlaunch digest:\t%s\n", manifest.TrustedMeasurement)
+	fmt.Fprintf(versionsWriter, "\tembedded reference values:\t%s\n", manifest.EmbeddedReferenceValuesJSON)
```

Comment:
This is imho a good use case for [`json.MarshalIndent`](https://pkg.go.dev/encoding/json#MarshalIndent).